django/contrib/admin/static/admin/css/nav_sidebar.css

```diff
@@ -0,0 +1,106 @@
+html, body {
+    height: 100%;
+}
```

Comment:
> I would expect this file to only contain the CSS necessary for the sidebar itself

These changes is required to make sidebar work properly, so this is why they placed in nav_sidebar.css which includes to template if sidebar is enabled. It looks OK to me that some component adds/overrides own/global CSS rules with its own CSS file.

Reply:
I understand that, but why not apply them to the existing CSS? I think we should so that the CSS between the sidebar and non-sidebar admin version remain closer in rendering logic such that it is less likely that tweaking one version will be a detriment to the other.

Reply:
The original idea here was to make sure if you disabled the navbar, everything is backwards compatible as it was before. However, I think you're right in that it'd be better to keep them as closely aligned as possible. I'll see what I can move over without breaking things for those with the sidebar disabled.

Related to this, since there's no longer any media queries, I've also moved the `<link>` tag further up the page, above the responsive CSS and above the extra styles block, so it will be easier for users to override these styles.

Reply:
This is done. Everything seems ok with and without, but there are a lot of combinations.